python/lsst/sims/utils/WcsUtils.py

```diff
@@ -1,103 +1,231 @@
-import numpy
+import numpy as np
```

Comment:
Because I didn't want to flood this PR with a bunch of small changes that meant nothing.  I adopted the philosophy "I will change whatever file I touch".  Once we have converged on this pull request, I will go through and change everything else.  That way, no one has to look at a cluttered file diff.